tool/tsh/tsh.go

```diff
@@ -819,3 +825,73 @@ func onShow(cf *CLIConf) {
 
 	fmt.Printf("Fingerprint: %s\n", ssh.FingerprintSHA256(pub))
 }
+
+// onStatus command shows which proxy the user is logged into and metadata
+// about the certificate.
+func onStatus(cf *CLIConf) {
+	tc, err := makeClient(cf, true)
+	if err != nil {
+		utils.FatalError(err)
+	}
+
+	proxyClient, err := tc.ConnectToProxy(cf.Context)
```

Comment:
why do you need to connect to the remote cluster for this command? I was thinking to local profile information for this purpose.

Reply:
Specifically, if client is logged into multiple proxies, we want to show this as well

Reply:
The users roles are not always available without connecting to the proxy. They are only available in the certificate for new style certificates.

Reply:
then don't show them for compat mode